array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,9 @@
+/* exported compact */
+function compact(array){
+  var newArray=[];
+  for (i = 0; i< array.length; i++) {
+    if (!!(array[i])  !== false) {
```

Comment:
You can actually simplify this conditional a lot. You can just do `if (array[i])` because conditonals will always either evaluate to true or false. So this would just check if each value is either "truthy" or "falsy"